### PR TITLE
Integrating smaller lantern-ui into flashlight-build

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

**Comment by oxtoacart** [comment]
Most of these changes were already reviewed [here](https://github.com/getlantern/lantern-ui/pull/82).  I'll call out what's new with specific comments.

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@ envvars.bash
 Lantern.app
 lantern.dmg
 src/github.com/getlantern/flashlight/ui/resources.go
+src/github.com/getlantern/lantern-ui/dist
```
Comment (oxtoacart) [inline .gitignore:21]: No longer checking in the dist folder.

File: genassets.bash [hunk 2]
```diff
@@ -2,8 +2,16 @@
 
 echo "Generating UI resources for embedding"
 
+echo "First, generating dist folder"
```
Comment (oxtoacart) [inline genassets.bash:5]: genassets now builds the dist folder each time it runs.

File: linuxcompile.bash [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (oxtoacart) [inline linuxcompile.bash:1]: Added script for compiling on Linux.

File: src/github.com/getlantern/lantern-ui/README.md [hunk 4]
```diff
@@ -75,6 +75,16 @@ config file (`config/compass.rb`):
...
 
+## Build
+
+Lantern UI uses [Gulp](http://gulpjs.com/) to build assets
+
+* `npm install --global gulp`
+* `npm install`
+* `gulp build`
+
+Embed 'dest' folder into production.
```
Comment (myleshorton) [inline src/github.com/getlantern/lantern-ui/README.md]: spelling of "dest"

Comment (oxtoacart) [inline src/github.com/getlantern/lantern-ui/README.md]: I didn't actually touch this, but it's fixed now.

**Comment by oxtoacart** [comment]
Retargeted - https://github.com/getlantern/flashlight-build/pull/69

**Pull request closed without merging**